operational/itam/schedule_itam_report/schedule-itam-report.pt

```diff
@@ -0,0 +1,192 @@
+name "Schedule ITAM Report"
+rs_pt_ver 20180301
+type "policy"
+short_description "Schedule a Flexera ITAM report (Custom view) and send it as a email to one or more recipients.  See the [README](https://github.com/flexera-public/policy_templates/tree/master/operational/itam/schedule_itam_reports) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
+long_description ""
+severity "medium"
+category "Operational"
+default_frequency "weekly"
+info(
+  version: "0.1.0",
+  provider: "Flexera ITAM",
+  service: "",
+  policy_set: "Schedule Flexera ITAM Report"
+)
+
+parameter "param_report_id" do
+  type "string"
+  label "Flexera ITAM Report ID"
+  description "Flexera ITAM Custom View Report ID"
+end
+
+parameter "param_report_title" do
+  type "string"
+  label "Report Title in the Mail"
+  default "Flexera ITAM Report"
+  description "Report header in the report result (e.g. Flexera ITAM Report)"
+end
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses of the recipients you wish to notify"
+  description "A list of email addresse(s) to notify"
+  default []
+end
+
+credentials "auth_flexera" do
+  schemes "oauth2"
+  label "flexera"
+  description "Select Flexera One OAuth2 credentials"
+  tags "provider=flexera"
+end
+
+pagination "itam_pagination" do
+  get_page_marker do
+    body_path jq(response, 'try(.nextPage | split("skipToken=") | .[1] | rtrimstr("%3D") | .+"=") catch null') # The skipToken is url encoded, so we need to decode the %3D to = .  This is not ideal implementation this seems to be enough without a true urldecode function
+  end
+  set_page_marker do
+    query "skipToken"
+  end
+end
+
+# /fnms/v1/orgs/{orgId}/reports
+datasource "fnms_report_index" do
```

Comment:
use `ds_` prefix for datasource

Reply:
Removed datasource in https://github.com/flexera-public/policy_templates/pull/1226/commits/1d884c09d545674513d743ac53271dd1f0c0e544